src/lyapunovs.jl

```diff
@@ -42,11 +42,9 @@ function lyapunovs(ds::DiscreteDS, N::Real; Ttr::Int= 100)
     J = jac(u)
     K = J*Q
 
-    F = qrfact(K)
-    Q .= F[:Q]
-    A_mul_signB!(Q, F[:R]) #ensure QR gives positive diagonal to R
```

Comment:
But when I was testing this, the re-configuration of `Q` was also necessary, because otherwise the step `K = J*Q` that happens before was not working, not giving the correct results...
Hm this is extremely weird that it works, since I can see all tests passing.

my mind is so confused right now

I will merge this non the less, just a final comment: the reason there is still some room left for improvement here is because the package `StaticArrays` does not have a method for qr-decomposition yet...